.github/workflows/build.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           java-version: 11
 
       - uses: gradle/wrapper-validation-action@v1
-      - run: ./gradlew build
+      - run: ./gradlew check build
```

Comment:
`build` already runs `check`

Reply:
🤔 why did the tests fail then when I ran them? I will double check what could be different. CI should've caught it, but maybe my OS/JVM is triggering something.

Reply:
I assume because you're on Windows and the fix only targets Windows users. CI runs on Linux. I run Mac OS.

You can confirm `build` includes `check` with `--dry-run`.

Reply:
Rebased on trunk and dropped commit. I'll check why this happened, I suspect git crlf handling right now.

Reply:
Yep `core.autocrlf=true` was set globally, changed it to `false` now. Btw, this wasn't just a test-fix, it's actually adding Windows support to this program.